### ISSUE TITLE
Not all zoom levels updating on re-render

### ISSUE THREAD

**Issue by KeithDeRuiter** [overviewer/Minecraft-Overviewer#1727]
I've rendered a map of my overworld using normal lighting with 4 copies for the four compass directions.  After I added a building to the overworld, I re-rendered the map and all of the views updated to show the new structure, but the "upper left" view, which is also first in the config file, appears to not reflect the new structure on its two most zoomed inlevels.  The new structure does show on the further out views, and on all three other views (the other three directions: upper right, lower left, and lower right).  The command was run with the --forcerender option in an attempt to make everything re-render, but that did not have any affect.

The following Overviewer version was used, compiled/run from cloned source:
Minecraft Overviewer 0.15.2 (751ba39)
built on Sun Feb  9 15:09:13 2020

The map is running in a vanilla 1.15.2 server, whose latest backup from a tar is unpacked and used for rendering (vs. on the live world folder).

The attached screenshots show the structure (an acacia hut and a test pillar with sand on top) at two zoom levels, and then present in another orientation at the same closer zoom level.

Upper left orientation with structure:
<img width="185" alt="UL-with-structure" src="https://user-images.githubusercontent.com/1069615/74624479-76e6b300-50fd-11ea-9662-cb53ad7b995e.PNG">

Upper left zoomed in showing the same hut/silly pillar missing:
<img width="313" alt="UL-structure-missing" src="https://user-images.githubusercontent.com/1069615/74624481-78b07680-50fd-11ea-86df-f0c6d4e06ede.PNG">

Lower right at the same close zoom level:
<img width="304" alt="LR-with-structure" src="https://user-images.githubusercontent.com/1069615/74624485-7bab6700-50fd-11ea-8664-77bb8d2b78fd.PNG">

Config file used for rendering:
[overviewer.txt](https://github.com/overviewer/Minecraft-Overviewer/files/4211846/overviewer.txt)

**Comment by enaut** [comment]
just to make sure... did you clear your caches? sometimes things are still cached in browser cache... [shift]+click on reload should do this.

**Comment by KeithDeRuiter** [comment]
Hmm, I don't know for sure, but I checked again it this morning and it's working as intended.  I feel like I did a reasonable amount of troubleshooting yesterday but apparently missed something and I don't recall if I checked with a clear cache.  Sounds like a likely culprit.  I'll rerun it this afternoon and test to see if the behavior is the same and resolved specifically by cache clearing.  If so, I'll update and close the issue!
